rocketpy/rocket/rocket.py

```diff
@@ -1174,6 +1170,33 @@ def add_parachute(
         self.parachutes.append(parachute)
         return self.parachutes[-1]
 
+    def add_sensor(self, sensor, position, x_position=0, y_position=0):
```

Comment:
> EDIT: Acctually, I think you really meant "position"... In that case I suggest you to clearly state that "this position is with respect to the user defined coordinate system".

The position is not relative to the user coordinate system though. It works just like the `eccentricity` from the other Rocket methods. I just thought that `position` was a more intuitive name then `eccentricity`

Do you think it would be best to have `x_eccentricity` and `y_eccentricity`?

Reply:
I think "position" is definitely ambiguous, we should try elaborating better names.

`eccentricity` or `offset` works for me, but I 

Other option is to receive a tuple of two values. Like this: `offset=(x, y)`